examples/src/agents/streaming.ts

```diff
@@ -4,7 +4,7 @@ import { BaseCallbackHandler } from "langchain/callbacks";
 import { ChatOpenAI } from "langchain/chat_models/openai";
 import { Calculator } from "langchain/tools/calculator";
 import { AgentAction } from "langchain/schema";
-import { Serialized } from "langchain/load/serializable";
+import { Serialized } from "@langchain/core/load/serializable";
```

Comment:
Manually did these, also forced me to push a change to the code block script. Same issue as runnables, double export w/ langchain & core so code blocks for this will reference the api refs from langchain, not core.